packages/circuit-ui/components/Input/Input.spec.tsx

```diff
@@ -102,11 +102,6 @@ describe('Input', () => {
     expect(actual).toMatchSnapshot();
   });
 
-  it('should render with inline styles when passed the inline prop', () => {
-    const actual = create(<Input inline label="Label" noMargin />);
-    expect(actual).toMatchSnapshot();
-  });
```

Comment:
> Did you check roughly how much usage of this we have?

I found only 3 usages in the dashboard.

There's the [`InlineElements`](https://circuit.sumup.com/?path=/docs/layout-inlineelements--three-inline-elements) component — do you wanna promote its use?

Reply:
I didn't even know we had this 🙈 🙈 🙈 So yeah maybe—happy to add some docs or a story for this pattern if you'd like

Reply:
That was a tongue-in-cheek question — I personally think the `InlineElements` component should be deprecated and removed 😅 If anything, this should be a style helper.

Reply:
Yeah a mixin might be more in line with what we've been doing recently. Do you prefer it in this case because it's easier to apply to any other component? (whereas the `InlineElements` component would often add a level to the DOM)?

 I'm also not opposed to just recommending custom styles (as in my snippet above), just thinking it'd make sense to document a recommended workaround to the `inline` prop

Reply:
>  Do you prefer it in this case because it's easier to apply to any other component? (whereas the InlineElements component would often add a level to the DOM)?

Exactly. The sole purpose of the component is stylistic, the HTML semantics don't matter. 

I'll go for custom styles for now.

Reply:
Done in 624673d.